x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/rules_table/bulk_actions.tsx

```diff
@@ -53,9 +58,23 @@ export const BulkActions: React.FC<BulkActionsProps> = React.memo(
             </EuiButtonEmpty>
           </EuiFlexItem>
         ) : null}
+        {showRetryFailedRulesButton ? (
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              iconType="refresh"
+              color={'warning'}
+              data-test-subj="reprocessFailedRulesButton"
+              onClick={reprocessFailedRules}
+              disabled={isTableLoading}
+              aria-label={i18n.REPROCESS_FAILED_ARIA_LABEL}
+            >
+              {i18n.REPROCESS_FAILED_RULES(numberOfFailedRules)}
+              {isTableLoading && <EuiLoadingSpinner size="s" />}
```

Comment:
NIT: `EuiButton` already has the `isLoading` prop that renders a spinner.

Reply:
done